safety-quiz/templates/register.html

```diff
@@ -23,27 +23,30 @@
         </div>
         <div class="form-group">
             <label for="email">Email address</label>
-            <input type="email" class="form-control" id="email" name="email" aria-describedby="emailHelp" value="{{ email }}" readonly required>
+            <input type="email" class="form-control-plaintext" id="email" name="email" aria-describedby="emailHelp" value="{{ email }}" readonly  required>
         </div>
         <div class="form-group">
             <label>Major</label>
-            <select class="form-control" id="major" name="major" required>
+            <select class="form-control" id="major" name="major" onchange="validateMName()" required>
                 <option selected value="">Choose your major...</option>
                 {% for each in majors %}
                 <option value="{{ each.id }}">{{ each.name }}</option>
                 {% endfor %}
                 <option value="-1">Other not listed.</option>
             </select>
+            <input class="form-control" type="text" id="custom-major" name="custom-major" hidden>
+
         </div>
         <div class="form-group">
             <label for="college">College</label>
-            <select class="form-control" id="college" name="college" required>
+            <select class="form-control" id="college" onchange="validateCName()" name="college" required>
```

Comment:
Minor change, could you expand the function name so it is easier to tell what the function does? "validateCollegeName" or "validateMajorName" is easier to understand than abbreviations

Reply:
Updated names